app/assets/javascripts/live_search.js

```diff
@@ -19,10 +19,11 @@
       this.saveState();
       this.$form.on('change', 'input[type=checkbox], input[type=text], input[type=radio]', this.formChange.bind(this));
 
-      this.$form.find('input[type=text]').keypress(
+      this.$form.find('input[type=text]').on('keypress blur',
         function(e){
-          if(e.keyCode == 13) {
+          if(e.keyCode == 13 || e.type == 'blur') {
```

Comment:
I find it easier to read things like keycodes if they've been put into a variable (e.g. `const ENTER_KEY = 13`). Not a required change though as it's reasonably clear already :)